synapseclient/api/json_schema_services.py

```diff
@@ -0,0 +1,175 @@
+import json
+from typing import TYPE_CHECKING, AsyncGenerator, Optional
+
+from synapseclient.api.api_client import rest_post_paginated_async
+
+if TYPE_CHECKING:
+    from synapseclient import Synapse
+    from synapseclient.models.mixins import (
+        InvalidJSONSchemaValidation,
+        JSONSchemaBinding,
+        JSONSchemaDerivedKeys,
+        JSONSchemaValidation,
+        JSONSchemaValidationStatistics,
+    )
+
+
+async def bind_json_schema_to_entity(
+    synapse_id: str,
+    json_schema_uri: str,
+    *,
+    enable_derived_annos: bool = False,
+    synapse_client: Optional["Synapse"] = None,
+) -> "JSONSchemaBinding":
```

Comment:
Is this the correct return types for these function calls? It seems like it's directly returning the call to `client.rest_*`, which is returning back a `Union[Dict[str, Any], str]` type

Reply:
This is not! I just made some edits.